src/domains/dashboard/components/head-main.tsx

```diff
@@ -261,11 +262,11 @@ export const HeadMain = ({
                 chartLibrary: "easypiechart",
                 title: "IPv6 Inbound",
                 units: "kbps",
-                aggrMethod: "sum",
                 width: "11%",
                 before: 0,
                 unitsCommon: "system.ipv6.mainhead",
                 ...(attributesOverrides && attributesOverrides["system.ipv6"]),
+                aggrMethod: mapDefaultAggrMethod("%"),
```

Comment:
I don't think the unit here are %

Reply:
good catch, I've updated it

Reply:
Many thanks @novykh ! 
However the problem is not fixed I think. I think we should be using hard coded strings.
With the latest commit you pushed, the aggMethod will return as "sum" whihc is the correct specified in the algorithm.
However I think we want "avg" here! 
So I propose we don't use the `mapDefaultAggrMethod` not only for performance reasons but more importantly because we want to hard code the aggrMethod too (like in that case)

Reply:
it was hard codes as sum before. What is the issue?

Reply:
I repeat it above in 3 comments: The way it is now it will also return "sum" which is wrong. There is no change from what it was in aggregation methods! (apart from unnecessarily calling `mapDefaultAggrMethod` which I propose it be removed)
If it's is not clear please give me a call.